rawfile/config/model.py

```diff
@@ -24,6 +24,34 @@
 from datetime import timedelta
 
 
+class StoragePool(BaseModel):
+    name: Annotated[
+        str, StringConstraints(strip_whitespace=True, min_length=3, max_length=63)
+    ] = Field(description="Name of the pool, Should be unique")
+    path: DirectoryPath = Field(description="Path of the pool, Should be unique")
+    reserved_capacity: (
+        ByteSize
+        | Annotated[
+            str,
+            StringConstraints(strip_whitespace=True, pattern=r"^\d+%$"),
+        ]
+    ) = Field(
+        default=ByteSize(0),
+        description="Reserves capacity of storage pool",
+    )
+    capacity_override: ByteSize | None = Field(
+        default=None,
+        description="Overrides total capacity of storage pool",
+    )
+    default_fs: FileSystemName = Field(
+        default=FileSystemName.EXT4,
+        description="Default filesystem used where creating volumes and fsType is not specified in storage class parameters",
+    )
+    is_default: bool = Field(
```

Comment:
Should this be a name to avoid setting `is_default` on more than 1 ?

Reply:
Yeah
changing moving it to global config and making it to be `defaultPool`